kratos/tests/cpp_tests/containers/test_flags.cpp

```diff
@@ -119,14 +119,11 @@ KRATOS_TEST_CASE_IN_SUITE(KratosFlagsSetMultiple, KratosCoreFastSuite) {
 }
 
 KRATOS_TEST_CASE_IN_SUITE(KratosFlagsEquality, KratosCoreFastSuite) {
-    Kratos::Flags flags1(INLET);
-    Kratos::Flags flags2(NOT_INLET);
 
-    KRATOS_CHECK_EQUAL( flags1 == flags2, false );
-    KRATOS_CHECK_EQUAL( flags1 != flags2, true );
-
-    KRATOS_CHECK_EQUAL( (!flags1) == flags2, true );
-    KRATOS_CHECK_EQUAL( flags1 == !flags2, true );
+    KRATOS_CHECK_EQUAL( INLET == NOT_INLET, false );
+    KRATOS_CHECK_EQUAL( INLET != NOT_INLET, true );
+    // trivial check (it is acutally 0 == 0) just to see that everything works when the first argument of == is not a lhs
+    KRATOS_CHECK_EQUAL( (~INLET & INLET) == NOT_INLET, true );
```

Comment:
This result of `(~INLET & INLET)` shouldn't be zero?

Reply:
zero with the inlet flag defined :man_facepalming: